bpf/lib/common.h

```diff
@@ -509,6 +510,19 @@ struct proxy6_tbl_value {
 	__u32 lifetime;
 } __attribute__((packed));
 
+/* ep_config corresponds to the EndpointConfig object in pkg/maps/configmap. */
+struct ep_config {
```

Comment:
@joestringer PTAL

Reply:
Apparently I missed that the MAC types are actually 8B in the tree, so all-up the non-pad fields below add to 44B. I guess we can round to the 8B border, so pad should be 4B rather than 1 as below.

Reply:
Fixed to be uint32 in both Cilium code and BPF.